primitives/blockchain/src/backend.rs

```diff
@@ -254,6 +254,8 @@ pub struct Info<Block: BlockT> {
 	pub finalized_hash: Block::Hash,
 	/// Last finalized block number.
 	pub finalized_number: <<Block as BlockT>::Header as HeaderT>::Number,
+	/// Number of concurrent leave forks
```

Comment:
```suggestion
	/// Number of concurrent leave forks.
```

(To align with the comments above.)